demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -3674,6 +3539,140 @@ export class DemoMeetingApp
     }
   }
 
+  private redirectFromAuthentication(quickjoin: boolean = false): void {
```

Comment:
I am bit confused as to why this method is named `redirectFromAuthentication`, can you please explain? It seems a bit vague. 

This method basically changes the application flow from `flow-authenticate` to `flow-devices` or `flow-meeting`.